### ISSUE TITLE
Wikipedia page about BOSC needs updating

### ISSUE THREAD

**Issue by nlharris** [OBF/bosc_materials#27]
The BOSC page (https://en.wikipedia.org/wiki/Bioinformatics_Open_Source_Conference) was last edited in 2020 and is out of date (for example, it has a link to the old wiki rather than our newer website). Does anyone know anyone who is empowered to edit it? I can suggest edits. Thanks!

**Comment by peterjc** [comment]
Anyone can edit a WikiPedia the page (unless wikipedia has banned them or their IP address), they have guidelines about conflicts of interest though:

https://en.wikipedia.org/wiki/Wikipedia:Conflict_of_interest

Some of your edits might be best as suggestions, but the URL update seems uncontentious so I did that just now.

**Comment by sudarsan2k5** [comment]
@nlharris can i work on this issue ??

**Comment by gedankenstuecke** [comment]
I've taken a small stab at adding some more information, but what would be really useful would be to maybe have a table with all past BOSC events and when/where they happened rather than this somewhat eclectic list!

**Comment by nlharris** [comment]
@gedankenstuecke luckily, such a list exists! see https://www.open-bio.org/events/bosc-2021/about/ (need to change future tense to past on that page) and my more complete internal spreadsheet, https://docs.google.com/spreadsheets/d/1eKKVcceKFqz0PeWxnzX4zcWKeppYIfDxrWeCFI9vaAg/edit#gid=0

**Comment by gedankenstuecke** [comment]
@nlharris Thanks, I've added that table!

**Comment by nlharris** [comment]
Thanks!

Can I request some other changes?

BOSC Logo: please replace with new logo
![Colour Icon](https://user-images.githubusercontent.com/348704/205401550-c900497f-9dc9-484c-bf45-95df45a14a3a.png)

> The Bioinformatics Open Source Conference (BOSC)

Could this be hyperlinked to https://www.open-bio.org/events/bosc?

> The Bioinformatics Open Source Conference (BOSC) is an [academic conference](https://en.wikipedia.org/wiki/Academic_conference) on [open-source programming](https://en.wikipedia.org/w/index.php?title=Open-source_programming&action=edit&redlink=1) in [bioinformatics](https://en.wikipedia.org/wiki/Bioinformatics)

The Bioinformatics Open Source Conference (BOSC) is an [academic conference] that
promotes and facilitates the [open source](https://en.wikipedia.org/wiki/open_source) development of [bioinformatics](https://en.wikipedia.org/wiki/bioinformatics) tools and open science.

> The BCC in 2020 took place online with two time schedules for eastern/western time zones

[BCC2020](https://bcc2020.github.io/) was held online, and the program was repeated twice a day for eastern/western time zones.

> The conference is held as a single track consisting of presentations, [poster sessions](https://en.wikipedia.org/wiki/Poster_session) and two [keynote talks](https://en.wikipedia.org/wiki/Keynote) by people of influence in open-source bioinformatics.[[1]](https://en.wikipedia.org/wiki/Bioinformatics_Open_Source_Conference#cite_note-bosc-2013-report-1)

The conference is held as a single track consisting of presentations, Birds of a Feather meetups, [poster sessions](https://en.wikipedia.org/wiki/Poster_session), sometimes a panel discussion, and two [keynote talks](https://en.wikipedia.org/wiki/Keynote) by people of influence in open-source bioinformatics.

Recent session topics have included:

- Open Science and Reproducible Research
- Open Biomedical Data
- Citizen/Participatory Science
- Standards and Interoperability
- Data Science
- Workflows
- Open Approaches to Translational Bioinformatics
- Developer Tools and Libraries
- Inclusion, Outreach and Training


> Since 2010, an informal two-day [Codefest](https://en.wikipedia.org/wiki/Codefest) has been held directly preceding the conference

Since 2010, an informal two-day [CollaborationFest](https://www.open-bio.org/events/bosc-2022/obf-bosc-collaborationfest/) has been held right before or after BOSC.

> As of November 2022, there have been 23 BOSC held around the world, of those 20 were purely in-person conferences, 2 purely remote due to the [COVID-19 pandemic](https://en.wikipedia.org/wiki/COVID-19_pandemic) and one that was organized as a hybrid meeting

As of November 2022, there have been 23 BOSCs held around the world. 20 were purely in-person conferences, 2 purely remote due to the [COVID-19 pandemic](https://en.wikipedia.org/wiki/COVID-19_pandemic) and one hybrid meeting ([BOSC 2022](https://www.open-bio.org/events/bosc-2022/)).

Thanks! 🙏

**Comment by nlharris** [comment]
@gedankenstuecke Are you the person who can make these updates, or is there a way for me to request them myself? (Does wikipedia have PRs?)

**Comment by gedankenstuecke** [comment]
In principle anyone can just live-edit the articles by clicking on the `Edit` tab on the top of the page and there's no pull requests or pre-publication review as any changes go live immediately (and can just be reverted by others if deemed appropriate). 

Having said that: I'm hesitant regarding the changes you suggested though, given that there's generally a policy of not editing pages about oneself/ones organizations due to conflicts of interest. There are some exceptions regarding referencing and I've tried to stay on that side of making edits that I felt towed that line. 

Some of the suggested edits here feel like they might reasonable be interpreted as falling more within marketing (e.g. adding direct links to our own websites within the body of the text rather than just using the referencing feature of MediaWiki as is the case for many of the suggested links so far). 

tl;dr: I've made some of the [smaller suggested edits here](https://en.wikipedia.org/w/index.php?title=Bioinformatics_Open_Source_Conference&diff=1128762324&oldid=1124270724) but don't want to fall on the wrong side of the Wikipedia policies for most of the other ones.

**Comment by nlharris** [comment]
Thanks, @gedankenstuecke. I understand your reluctance to make changes that might seem too self-promoting, so I won't push any of the edits I suggested except for changing the logo - I would love to have our new map-pear logo on the wikipedia page if that's possible and would not burn too much credibility capital for you.

**Comment by gedankenstuecke** [comment]
No problem! Happy to add the logo, do you have a reference for where the file is and under which license? :)

**Comment by nlharris** [comment]
Thanks! https://github.com/OBF/bosc_materials/blob/master/logos/2021/Colour%20Icon.png (or more generally, any of the files in https://github.com/OBF/bosc_materials/tree/master/logos/2021).

Re license, good question -- there should be a LICENSE fill in this repo but I don't see one. @cjfields can you advise on which license we should use? CC BY 4.0 (as in https://github.com/OBF/newsletter/blob/master/LICENCE)? (Is it really necessary to put the whole text of the license in the LICENSE file rather than just a URL?)

**Comment by gedankenstuecke** [comment]
I think CC BY SA would be good in principle, but it may be worth to check with the creators of the logos what they would be happy with (as I don't know how and under which circumstances they were made)

**Comment by peterjc** [comment]
While nice in principle given how we use the logo on fliers etc, does an
attribution clause make trouble for us? There is something to be said for
an all rights reserved approach to logos specifically (see eg Mozilla
Firefox logo/trademark/license versus Debian rebranding as Ice Weasel).
